src/app/config/thunks.js

```diff
@@ -120,3 +118,145 @@ export const createCollectionRequest = collection => (dispatch, getState) => {
             console.error(error);
         });
 };
+
+export const getUsersRequest = () => dispatch => {
+    users
+        .getAll({ active: true })
+        .then(response => {
+            dispatch(getUsersRequestSuccess(response));
+        })
+        .catch(error => {
```

Comment:
the try/catch here would be better at source in `getAll()` in `src/app/utilities/api-clients/user.js`